### PR TITLE
Implement Label

### LINKED ISSUE #145: Update FindCommand to use Label

### PR DESCRIPTION
**Pull request by HynRidge**
Implement Label to be used in TagCommand and FindCommand.
Add some testcases for FindCommand as well
Modify ParserUtil to parse Label and Set<Label>

### CODE REVIEW

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 1]
```diff
@@ -1,16 +1,45 @@
...
+import seedu.address.model.tag.TagName;
+
 
 
 /**
  * Contains utility methods for populating {@code AddressBook} with sample data.
  */
 public class SampleDataUtil {
     public static Tag[] getSampleTags() {
-        return new Tag[] {};
```
Comment (luo-git) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java:17]: This part should not be changed. This are the default tags when HelloFile is launched the first time. We might want to keep this list empty?

File: src/test/java/seedu/address/testutil/TagBuilder.java [hunk 2]
```diff
@@ -37,11 +39,12 @@ public TagBuilder() {
     public TagBuilder(Tag tagToCopy) {
         tagName = tagToCopy.getTagName();
         fileAddress = tagToCopy.getFileAddress();
+        labels = new HashSet<>(tagToCopy.getLabels());
```
Comment (luo-git) [inline src/test/java/seedu/address/testutil/TagBuilder.java]: Should there be an empty line here?

**Review by luo-git — APPROVED** [review]
Good job! Very clean code! Just two minor comments.

File: src/test/java/seedu/address/testutil/TagBuilder.java (hunk 2 above)
Comment (HynRidge) [reply src/test/java/seedu/address/testutil/TagBuilder.java]: what do you mean by empty line??

Comment (HynRidge) [reply src/test/java/seedu/address/testutil/TagBuilder.java]: Okay empty line deleted

File: src/main/java/seedu/address/model/util/SampleDataUtil.java (hunk 1 above)
Comment (HynRidge) [reply src/main/java/seedu/address/model/util/SampleDataUtil.java:17]: Okay amend it!

**Review by luo-git — APPROVED** [review]
LGTM!

**Pull request merged**